changelog.d/20240814_145223_30907815+rjmello.rst

```diff
@@ -0,0 +1,9 @@
+Deprecated
+^^^^^^^^^^
+
+- The ``HighThroughputEngine`` is now marked for deprecation. All users should
+  migrate to ``GlobusComputeEngine``.
+
+  To help with migration, we suggest checking out our many endpoint configuration
+  examples, all of which use ``GlobusComputeEngine``:
+  https://globus-compute.readthedocs.io/en/latest/endpoints/endpoint_examples.html.
```

Comment:
Suggest linking to stable, and perhaps not including the raw link, but rather as part of the text.  Not a biggie, but "looks nicer."  Perhaps:

```restructuredtext
  To help with migration, we suggest checking out our many `endpoint configuration
  examples <https://globus-compute.readthedocs.io/en/latest/endpoints/endpoint_examples.html>`_, all of which use ``GlobusComputeEngine``.
```

Reply:
Good call; I just pushed an update.